### PR TITLE
Fix missing image in "federated_cloud_sharing.rst"

### PR DESCRIPTION
**Pull request by danxuliu**
This adds back the file removed in commit 8106b6002509fba38c238996179d8c5f85442f24 but still used in [_federated_cloud_sharing.rst_](https://github.com/nextcloud/documentation/blob/acb907453dcd7c0ef946342b8a6202fdaf5ecbf1/user_manual/files/federated_cloud_sharing.rst#creating-a-new-federation-share).

The remote user shown in the image is `username@sub.example.com/nextcloud`, so the example name used in the
page did not match and thus was removed (it was not replaced as it felt redundant with the `username` already used in the previous sentence).

### CODE REVIEW

File: user_manual/files/federated_cloud_sharing.rst [hunk 1]
```diff
@@ -14,8 +14,7 @@ by default. Follow these steps to create a new share with other Nextcloud or own
 
 1. Go to your ``Files`` page and click the **Share** icon on the file or directory 
 you want to share. In the sidebar enter the username and URL of the remote user
-in this form: ``<username>@<oc-server-url>``. In this example, that is
-``layla@remote-server/nextcloud``. The form automatically echoes the address 
+in this form: ``<username>@<oc-server-url>``. The form automatically echoes the address
```
Comment (wiswedel) [inline user_manual/files/federated_cloud_sharing.rst:17]:
```suggestion
in this form: ``<username>@<oc-server-url>``. The form automatically confirms the address
```

"echo" is kind of techy. I'd substitute that with a more human word.

**Review by wiswedel — CHANGES_REQUESTED** [review]

File: user_manual/files/federated_cloud_sharing.rst (hunk 1 above)
Comment (danxuliu) [reply user_manual/files/federated_cloud_sharing.rst]: For me _confirms_ is less clear in this context than _echoes_... but I guess I am kind of techy :-P So done :+1:

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**